### DISCUSSION TITLE
Best way to deal with products that aren't offered any more

### DISCUSSION THREAD

**Discussion by philsward** [schemaorg/schemaorg#3002] [Ideas]
What is the best "Availability" to use for a product that isn't offered any more? Keep in mind the distinction between a product that has been discontinued because it isn't manufactured any longer and a product that isn't being carried any more but is still widely available.

From an SEO standpoint, it makes sense to keep a product listed on a site in the off-chance that a custom might look for a similar product (if suggestions are marketed well on the given site). But there's no way to relay this information to the aggregators like Google Merchant to shut them up about an invalid availability type.

Should there be an availability option such as "NoLongerOffered"?

Just curious what the best approach might be.

**Comment by WeaverStever** [comment]
These are the current enumerations for https://schema.org/availability


BackOrder
Discontinued
InStock
InStoreOnly
LimitedAvailability
OnlineOnly
OutOfStock
PreOrder
PreSale
SoldOut

see https://schema.org/ItemAvailability

(**Note that the url to the enumeration is expected**)
~~~~
  "offers": {
    "@type": "Offer",
    "availability": "https://schema.org/Discontinued",
    "price": "55.00",
    "priceCurrency": "USD"
  },
  ~~~~

**Comment by WeaverStever** [comment]
You can also combine the enumerations, I don't know what you would do to suggest a similar product.

~~~~
  "offers": {
    "@type": "Offer",
    "availability": ["https://schema.org/Discontinued", "https://schema.org/SoldOut"],
    "price": "55.00",
    "priceCurrency": "USD"
  },
~~~~

**Comment by philsward** [comment]
Thanks @WeaverStever 

I didn't know about combining them. I'm not sure how well Google would accept that, though it might work fine in other areas.

Thinking about "SoldOut", it probably is the best route for this scenario. It leaves the door open to possibly offering it again, but there is no guarantee. I guess I've never really thought about it that way because I just always viewed SoldOut and OutOfStock as saying the same thing using different words.

I will say, if there was a NoLongerOffered enumeration, it would present a definitive about the status instead of leaving it open-ended. I can see where customers will call in asking when it will be available again even with SoldOut as the enumeration. Then again, what I send to Google (SoldOut) vs what I portray to the customer "No Longer Offered" would be fine. It just makes my life a little more complicated.

**Comment by WeaverStever** [comment]
@philsward 

I think you are right about SoldOut, probably the best way to get your customer inquiries about restocking / alternate products.

**Comment by WeaverStever** [comment]
IMHO 

SoldOut = We are not planning to order anymore
OutOfStock = We will continue taking orders, but there will be a longer wait
